### PR TITLE
chore: refine the refactored branch codes

### PR DESCRIPTION
**Pull request by will-2012**
### Description

Refine the refactored branch codes.

### Rationale

N/A.

### Example

N/A.

### Changes

Notable changes: 
* signer: polish typo.
* gater: refine codes and add router ut.

### CODE REVIEW

File: modular/gater/gater.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func (g *GateModular) Start(ctx context.Context) error {
 
 func (g *GateModular) server(ctx context.Context) {
 	router := mux.NewRouter().SkipClean(true)
-	if g.baseApp.EnableMetrics() {
+	if g.baseApp.IsEnableMetrics() {
```
Comment (actioncli) [inline modular/gater/gater.go:45]: enable is betweer to isenable?

File: modular/gater/router.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ const (
 	listBucketReadRecordRouterName   = "ListBucketReadRecord"
 	requestNonceName                 = "RequestNonce"
 	updateUserPublicKey              = "UpdateUserPublicKey"
-	queryUploadProgressRouterName    = "queryUploadProgress"
+	queryUploadProgressRouterName    = "QueryUploadProgress"
```
Comment (actioncli) [inline modular/gater/router.go:25]: big is correct to go-sdk?

File: modular/gater/helper_test.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	testDomain = "www.route-test.com"
+	gw         = &GateModular{
+		domain: testDomain,
+	}
+	scheme     = "https://"
+	bucketName = "test-bucket-name"
+	objectName = "test-object-name"
+)
+
+func setupRouter(t *testing.T) *mux.Router {
```
Comment (actioncli) [inline modular/gater/helper_test.go:19]: move to router_test.go file?

File: modular/gater/gater.go (hunk 1 above)
Comment (will-2012) [reply modular/gater/gater.go]: Fixed.

File: modular/gater/router.go (hunk 2 above)
Comment (will-2012) [reply modular/gater/router.go:25]: Yes, it is only used in the gater.

File: modular/gater/helper_test.go (hunk 3 above)
Comment (will-2012) [reply modular/gater/helper_test.go]: Fixed.

**Review by joeylichang — APPROVED** [review]

**Pull request merged**